pkg/agent/flowexporter/connections/conntrack_ovs.go

```diff
@@ -189,9 +226,20 @@ func flowStringToAntreaConnection(flow string, zoneFilter uint16) (*flowexporter
 	conn.IsActive = true
 	conn.DoExport = true
 
+	klog.V(2).Infof("Convert flow string: %v into connection: %+v", flow, conn)
```

Comment:
s/Convert/Converted

Verbosity 2 is definitely too high here, maybe use 5 (https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md)

Reply:
Thanks, done.